src/main/java/uk/ac/ebi/ena/webin/cli/WebinCliParameters.java

```diff
@@ -28,6 +28,7 @@
   private File outputDir;
   private File inputDir = new File(".");
   private String centerName;
+  private String enaSubmissionTool;

```

Comment:
- Please change the type from String to WebinSubmissionTool (EnaSubmissionTool). There is no need to support an unconstrained, and possibly wrong, submission tool. Much better just to have the enum in the WebinCliParameters.
- Please default the value to WEBIN-CLI (suggested other enumeration value to be added).